velox/exec/RowContainer.h

```diff
@@ -127,10 +131,23 @@ class RowContainer {
   // Allocates a new row and initializes possible aggregates to null.
   char* newRow();
 
+  uint32_t rowSize(const char* row) const {
+    return fixedRowSize_ +
+        (rowSizeOffset_
+             ? *reinterpret_cast<const uint32_t*>(row + rowSizeOffset_)
```

Comment:
reinterpret_cast<const uint32_t*>(row + rowSizeOffset_) is long and repeats multiple times. It would be nice to put it in a helper function.